app/src/sharedTest/java/org/simple/clinic/TestData.kt

```diff
@@ -1071,7 +1071,9 @@ object TestData {
       registrationFacilityUuid = registrationFacilityUuid,
       status = status,
       createdAt = createdAt,
-      updatedAt = updatedAt
+      updatedAt = updatedAt,
+      capabilities = null,
+      teleconsultPhoneNumber = "+919995558883"
```

Comment:
Let's provide the option to override these values from `ongoingLoginEntry` method params or else it would be a hidden behaviour.